### PR TITLE
chore(wasm): benchmark server keys and format javascript code

### PR DESCRIPTION
**Pull request by soonum**
This contains several commits to make changes more concise.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -153,6 +153,13 @@ check_actionlint_installed:
 fmt: install_rs_check_toolchain
 	cargo "$(CARGO_RS_CHECK_TOOLCHAIN)" fmt
 
+.PHONY: fmt_js # Format javascript code
+fmt_js:
```
Comment (IceTDrinker) [inline Makefile:157]: needs an nvm install ?

File: Makefile [hunk 2]
```diff
@@ -167,6 +174,13 @@ check_fmt_gpu: install_rs_check_toolchain
 	cargo "$(CARGO_RS_CHECK_TOOLCHAIN)" fmt --check
 	cd "$(TFHECUDA_SRC)" && ./format_tfhe_cuda_backend.sh -c
 
+.PHONY: check_fmt_js # Check javascript code format
+check_fmt_js:
```
Comment (IceTDrinker) [inline Makefile:178]: same thing

File: tfhe/web_wasm_parallel_tests/Makefile [hunk 3]
```diff
@@ -23,3 +23,13 @@ bench: run_server
 
 .PHONY: bench-ci # Run benchmarks on web client in CI
 bench-ci: run_server
+
+.PHONY: fmt # Format Javascript code
+fmt:
```
Comment (IceTDrinker) [inline tfhe/web_wasm_parallel_tests/Makefile:28]: needs npm installed ? don't know if it's already installed in here

Comment (soonum) [reply tfhe/web_wasm_parallel_tests/Makefile:28]: At this point we consider `npm` has being installed. Look at the `run_server` recipe in this file.

Comment (IceTDrinker) [reply tfhe/web_wasm_parallel_tests/Makefile:28]: ok thx

**Review by IceTDrinker — APPROVED** [review]
Thanks !

**Review by IceTDrinker — APPROVED** [review]

**Review by IceTDrinker — APPROVED** [review]

**Pull request merged**